src/org/zaproxy/zap/view/SessionIncludeInProxyPanel.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2010 psiinon@gmail.com

```

Comment:
This should be set as 2016 and "the ZAP Development Team" (check other recent PRs if you want to copy/paste)